tools/slang-cpp-extractor/slang-cpp-extractor-main.cpp

```diff
@@ -193,6 +195,10 @@ class Node : public RefObject
 
         /// True if has a derived type that is reflected
     bool hasReflectedDerivedType() const;
+        /// Stores in out any reflected derived types
+    void getReflectedDerivedTypes(List<Node*>& out) const;
+
+    static void filterReflectedClassLike(List<Node*>& io);
```

Comment:
Nits:

Why not `outDerivedTypes` instead of just `out` for the `getReflectedDerivedTypes` parameter?

Similarly, could `io` be changed to something more descriptive? The function name/signature isn't very clear here (probably needs a comment). It took me a while to come to the conclusion that this operation probably takes as input the list of nodes `io`, and filters it down to just those nodes that represent "class-like" nodes, returning the resulting filtered list back in `io`.